hadoop-yarn-project/hadoop-yarn/hadoop-yarn-common/src/main/java/org/apache/hadoop/yarn/security/client/ClientToAMTokenIdentifier.java

```diff
@@ -50,31 +50,25 @@ public ClientToAMTokenIdentifier() {
 
     public ClientToAMTokenIdentifier(ApplicationAttemptId id, String client) {
         this();
```

Comment:
still not needed

Reply:
still not part of this PR but from original implementation of ClientToAMTokenIdentifier (line 51)